openpype/hosts/resolve/startup.py

```diff
@@ -0,0 +1,40 @@
+import os
+
+# Importing this takes a little over a second and thus this means
+# that we have about 1.5 seconds delay before the workfile will actually
+# be opened at the minimum
+import openpype.hosts.resolve.api
+
+
+def launch_menu():
+    from openpype.pipeline import install_host
+    print("Launching Resolve OpenPype menu..")
+
+    # Activate resolve from openpype
+    install_host(openpype.hosts.resolve.api)
+
+    openpype.hosts.resolve.api.launch_pype_menu()
+
+
+def open_file(path):
+    # Avoid the need to "install" the host
+    openpype.hosts.resolve.api.bmdvr = resolve  # noqa
```

Comment:
I am not sure about this. This seems to be quite hacky solution. Or can you explain more why do we need to do  it this way?

Reply:
I mostly _tried_ to speed up the launchijng as much as possible since it's relatively slow to trigger the install of the Resolve host and Resolve's startup UI is active during this time. So a user might be inclined to click to open a project even though the "open last workfile" bit might still.

As mentioned in a comment above even the importing of the python files alone took about 1.5 seconds for me (which might be due to the machine testing it from isn't running the code from an SSD)

Reply:
I have now cleaned this up logically with a slower runtime as a result (so it'll take a bit longer before the workfile opens) but at least the code is somewhat simplified. Let me know how it works for you @jakubjezek001